api/users/pagination.py

```diff
@@ -0,0 +1,60 @@
+from collections import OrderedDict
+from django.core.urlresolvers import reverse
+
+from api.base.pagination import JSONAPIPagination
+from rest_framework.response import Response
+
+class UserNodeLogPagination(JSONAPIPagination):
+
+    node_log_aggregates = None
+
+    def get_paginated_response(self, data):
+        """
+        Formats paginated response in accordance with JSON API.
+
+        Creates pagination links from the view_name if embedded resource,
+        rather than the location used in the request.
```

Comment:
Change docstring on method instead of copying it - I was confused and thought you overrode this method to handle embedded. But it seems like you just needed a place to put aggregates/last_log_date info.